README.md

```diff
@@ -1,33 +1,25 @@
+[![](https://img.shields.io/static/v1?label=maven-central&message=0.4.5&color=blue)](https://search.maven.org/artifact/io.github.ztkmkoo/dss-server/0.4.5/jar)
 [![Build Status](https://travis-ci.org/ztkmkoo/dss.svg?branch=develop)](https://travis-ci.org/ztkmkoo/dss.svg?branch=develop)
 [![Coverage Status](https://coveralls.io/repos/github/ztkmkoo/dss/badge.svg)](https://coveralls.io/github/ztkmkoo/dss)
 [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/distributed-server-system/dss)
+
 [![Bugs](https://sonarcloud.io/api/project_badges/measure?project=ztkmkoo_dss&metric=bugs)](https://sonarcloud.io/dashboard?id=ztkmkoo_dss)
 [![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=ztkmkoo_dss&metric=code_smells)](https://sonarcloud.io/dashboard?id=ztkmkoo_dss)
 [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=ztkmkoo_dss&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=ztkmkoo_dss)
+
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=ztkmkoo_dss&metric=alert_status)](https://sonarcloud.io/dashboard?id=ztkmkoo_dss)
 [![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=ztkmkoo_dss&metric=security_rating)](https://sonarcloud.io/dashboard?id=ztkmkoo_dss)
 [![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=ztkmkoo_dss&metric=vulnerabilities)](https://sonarcloud.io/dashboard?id=ztkmkoo_dss)
 
-
-
 ## DSS
 
-`It is a start project. Not released yet!!`
-
 `DSS` is a distributed server system which enables quick and easy development of clustered server applications. It is dependency on netty network module and akka multi-thread and cluster module.
 
-## CONTRIBUTION
-
-https://github.com/ztkmkoo/dss/blob/develop/CONTRIBUTING.md
-
-## CODE CONVENTION
-
-https://github.com/ztkmkoo/dss/blob/develop/CODE_CONVENTION.md
-
-## CODE OF CONDUCT
-
-https://github.com/ztkmkoo/dss/blob/develop/CODE_OF_CONDUCT.md
-
-## TUTORIAL
+## GUIDE
 
-https://github.com/ztkmkoo/dss/wiki/Project-Tuturial(KR)
\ No newline at end of file
+- [Project Structure](https://github.com/ztkmkoo/dss/wiki/Project-Structure)
+- [Code Convention](https://github.com/ztkmkoo/dss/wiki/Code-Style)
+- [Tutorial](https://github.com/ztkmkoo/dss/wiki/Project-Tuturial)
```

Comment:
It seems that there is no project-tutorial page, is right?

Reply:
I agree. need to change it to https://github.com/ztkmkoo/dss/wiki/Project-Tutorial(KR), I guess.

Reply:
Yap.. i see,
i create https://github.com/ztkmkoo/dss/wiki/Dss-REST-Tuturial this one.
Thanks~

Reply:
``` suggestion
- [Project Structure](https://github.com/ztkmkoo/dss/wiki/Project-Structure)
- [Code Convention](https://github.com/ztkmkoo/dss/wiki/Code-Style)
- [Tutorial](https://github.com/ztkmkoo/dss/wiki/Dss-REST-Tuturial)
```